chrome_gpu_bug_count.sh

```diff
@@ -7,7 +7,7 @@ BASEDIR=$(dirname "$0")
 INPUT="$BASEDIR/chrome_gpu_triage.json"
 SCRIPT="$BASEDIR/issue_counter.py"
 mkdir -p "$OUTDIR"
-OUTFILE="$OUTDIR/$(date --iso-8601).json"
+OUTFILE="$OUTDIR/$(date "+%Y-%m-%d-%H:%M").json"
```

Comment:
I think I updated the PR with the additional commit correctly? I don't GitHub much, so if I need to rewrangle, let me know.